pypesto/sampling/adaptive_metropolis.py

```diff
@@ -38,6 +38,8 @@ def default_options(cls):
             'cov0': None,
             # target acceptance rate
             'target_acceptance_rate': 0.234,
+            # show progress
+            'show_progress': True,
```

Comment:
What about rename to `progress_bar`. It is more specific.

Reply:
Both fine for me. I thought maybe at some point one there may be other "progress indicators" aside from the progressbar, and usually one wants to switch either all on or all off.